aws_batch/indexing.py

```diff
@@ -1,148 +0,0 @@
-#!/usr/bin/env python
```

Comment:
we might still need this?

Reply:
Ok I've put it back in this repo while also leaving it in idseq-pipeline. I do think it will be good to just make it a part of idseq-pipeline because eventually we want to update index regularly and automatically. Then the indexing script and the pipeline scripts should have identical versions (e.g. say, we switch to a new release of GSNAP). So it makes sense to have them in the same package